client/src/components/ReviewsPhrases.jsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
+import styled from 'styled-components';
 
 const ReviewsPhrases = (props) => {
   console.log('props phrases : ', props.phrases)
   const phrases = props.phrases ? props.phrases : [];
-  console.log('props phrases : ', props.phrases)
+  console.log('props phrases : ', props.phrases);
```

Comment:
Consider deleting console.log. `Command + shift + f` in VScode allows you to do a global search. Type console.log and you can go through all the files to delete the ones you don't need in production code.

Reply:
Agree.  I will look into it for next PR.